inference-engine/src/vpu/common/include/vpu/utils/containers.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <algorithm>
+
+#include "error.hpp"
+
+namespace vpu {
+
+template<class Key, class Value, template<class...> class Map>
+inline std::vector<Key> getKeys(const Map<Key, Value>& map) {
+    auto keys = std::vector<Key>{};
```

Comment:
Just curious, what is the purpose of such variable creation syntax.
`std::vector<Key> keys;` is shorter and more common way

Reply:
There is no much functional details about that, just code style. You can see some motivation at Herb Sutter's talk on cppcon about that.